index.html

```diff
@@ -42,5 +45,169 @@ <h2>LET'S CONNECT</h2>
+    <li>2015</li>
+</ol>
+
+<p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+<ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javaScipt</li>
+</ul>
+<button type="button" class="button"> See Project</button>
+</div>
+
+
+
+<div class="project">
+    <img src="Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1">
+    <h2>Multi-Post <br>Stories</h2>
+    
+ <ol>
+        <li>CANOPY</li>
+        <li><img src="Images/Counter.png" alt="Counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="Images/Counter.png" alt="Counter"></li>
+        <li>2015</li>
+ </ol>
+
+ <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+ <ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javaScipt</li>
+</ul>
+
+<button type="button" class="button"> See Project</button>
+</div>
+
+
+<div class="project">
+    <img src="Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2">
+
+    <h2>Tonic</h2>
+
+    <ol>
+        <li>CANOPY</li>
+        <li><img src="Images/Counter.png" alt="Counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="Images/Counter.png" alt="Counter"></li>
+        <li>2015</li>
+    </ol>
+    
+    <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+    
+    <ul>
+        <li>html</li>
+        <li>css</li>
+        <li>javaScipt</li>
+    </ul>
+    <button type="button" class="button"> See Project</button>
+
+</div>
+
+
+<div class="project">
+    <img src="Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3">
+    <h2>Multi-Post <br>Stories</h2>
+    
+ <ol>
+        <li>CANOPY</li>
+        <li><img src="Images/Counter.png" alt="counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="Images/Counter.png" alt="counter"></li>
+        <li>2015</li>
+ </ol>
+
+ <p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+ <ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javaScipt</li>
+</ul>
+
+<button type="button" class="button"> See Project</button>
+
+</div>
+</section>
+
+
+
+<section class="about">
+    <h2>About <br> Myself</h2>
+
+    <p>Hello I'm a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience!If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me. </p>
+
+   <h3>LET'S CONNECT</h3>
+
+<ul class="icons">
+    <li>
+        <a href="#"><img src="images/Vector.png" alt="Vector"></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Linkedinicon.png" alt="Linkedin icon"></a>
+    </li>
+
+    <li>
+        <a href="#"><img src="images/Vector1.png" alt="Vector 1"></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Vector2.png" alt="Vector 2"></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Vector3.png" alt="Vector 3"></a>
+    </li>
+    
+</ul>
```

Comment:
please try to fix the icons links maybe they are working on your machine but not in others so Kindly fix them so I can see them 
to make them show up please change your src to `src="./Images/(name of the image )"`